types/office-js-preview/index.d.ts

```diff
@@ -14887,27 +14908,40 @@ declare namespace Office {
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose or Read
          *
+         * **Important**:
+         *
+         * - {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

Comment:
It looks like it will read "Personal contact groups added as appointment locations aren't returned by this method." I can't quite parse this sentence. What are the groups added to? Is there a suppressed helper verb with "added" ("were added", "will be added")? Does "as" in this sentence mean "because"?

Reply:
> Does this iteration help clarify the sentence: "The `getAsync` method doesn't return personal contact groups that were added to the **Location** field of an appointment."?

Yes. That's much better.